routing/index_graph_starter.cpp

```diff
@@ -189,6 +189,15 @@ void IndexGraphStarter::RedressRoute(vector<Joint::Id> const & route,
     Joint::Id const prevJoint = route[i];
     Joint::Id const nextJoint = route[i + 1];
 
+    // If a route goes through a loop-edge starting from a joint and finishing
```

Comment:
Are you sure that it's a good idea to continue here? The number of elements in routePoints will be smaller than it was.